packages/quantic/force-app/main/default/lwc/quanticDocumentSuggestion/__tests__/quanticDocumentSuggestion.test.js

```diff
@@ -0,0 +1,371 @@
+jest.mock('c/quanticHeadlessLoader');
+
+import QuanticDocumentSuggestion from 'c/quanticDocumentSuggestion';
+import {buildCreateTestComponent, cleanup, flushPromises} from 'c/testUtils';
+import * as quanticHeadlessLoader from 'c/quanticHeadlessLoader';
+const headlessLoaderMock = jest.mocked(quanticHeadlessLoader);
+
+const engineMock = {
+  id: 'exampleEngineIdMock',
+  dispatch: jest.fn(),
+};
+headlessLoaderMock.initializeWithHeadless.mockImplementation(
+  async (element, _, initialize) => {
+    initialize(engineMock);
+  }
+);
+headlessLoaderMock.getHeadlessEnginePromise.mockImplementation(() =>
+  Promise.reject({message: 'Skip initialization'})
+);
```

Comment:
This is causing the following which is polluting the logs: https://github.com/coveo/ui-kit/actions/runs/14365598472/job/40277925597?pr=5146
<img width="1030" alt="Screenshot 2025-04-11 at 7 59 28 AM" src="https://github.com/user-attachments/assets/b1942a2c-455c-441f-9d17-aa1c2235f207" />

We need to address this, a suggestion could be to handle initialization like we do for other components.

Reply:
Le console error est logged par le component de quickview par contre mais je suis d'accord que ca gosse.. Vue que ca utilise `getHeadlessEnginePromise` a la place de `initializeWithHeadless`. Sinon je dois mocker tout le reste (getHeadlessBundle, buildQuickView etc.) ou j'enleve le console.error 🤷‍♂️